cmd/createcluster.go

```diff
@@ -202,7 +214,7 @@ func runCreateCluster(ctx context.Context, w io.Writer, conf clusterConfig) erro
 		writeWarning(w)
 	}
 
-	writeOutput(w, conf.SplitKeys, conf.ClusterDir, numNodes)
+	writeOutput(w, conf.SplitKeys, conf.ClusterDir, numNodes, len(conf.KeymanagerAddrs) != 0)
```

Comment:
suggest creating a variable above: `keysToDisk := len(conf.KeymanagerAddrs) == 0`

Reply:
you do this check twice in the same function, "len(conf.KeymanagerAddrs) != 0", rather create a local variable in the function and use that variable

Reply:
done